### ISSUE TITLE
[REQUEST] Replace Windows Subsystem Screenreader with the latest TalkBack version (14 or the Nik Addon for 13), only if you can make it capture the keyboard shortcuts. If you can't, allow the current Windows Subsystem Screenreader to use any text to speech engine

### ISSUE THREAD

**Issue by CiprianDinca9** [MustardChef/WSABuilds#182]
### Describe your request | 描述您的要求

For you to understand this matter, you need to first know how TalkBack works if installed from the Google PlayStore. Yes, it'll speak. But any talkback keyboard shortcut (see them in talkback settings/keyboard shortcuts) aren't accepted. The same happens also when installing Jieshuo, https://github.com/nirenr/jieshuo. So whenever we install and use any of these screen readers, keyboard shortcuts, whatever they might be, aren't accepted and therefore the screenreader can only be used with a mouse/trackpad, which defeats the purpose. I'd only guess that a touch screen monitor would help, but that's not the scope of this post. If you can make the WSA accept keyboard shortcuts from any screen reader, then load it with Jieshuo linked in this post and make it by default when pressing windows+control+T. If you can't make Jieshuo receive keyboard input properly, then try it with TalkBack, perhaps the Nik GApps TalkBack addon would be something.
If even that doesn't work, then at least make it so we can change the TTS engine to the current subsystem screenreader. What happens? Selecting a TTS engine from Text to Speech output won't change the speaking TTS engine for the screenreader. Although, I really hope you can make these screen readers capture keyboard shortcuts Please try! The current screenreader is based on TalkBack9, which is old, slow, plus Windows OneCore voices are unresponssive. Please kindly try fixing this issue! I tried sending an E-mail to Microsoft to look into this matter, but they haven't replied or let me know of anything. So my only hope is actually you.
Please kindly help us!

**Comment by sadtango289** [comment]
Forgive me for the brief presentation and not make new post because I'm not a professional > What is bug ?
Please check screenshot https://i.imgur.com/KIrWOJN.png
